src/guides/v2.3/install-gde/trouble/php/tshoot_install-main.md

```diff
@@ -17,5 +17,4 @@ functional_areas:
 *  [During installation, exception SessionHandler::read()](https://support.magento.com/hc/en-us/articles/360033427272)
```

Comment:
Delete this topic. It's not linked to the TOC and is of questionable value.

Reply:
Done